pisa/core/events_pi.py

```diff
@@ -66,6 +66,25 @@
 )
 
 
+
+def append_arrays_dict(key, val, sdict):
+    '''
+    Helper function for appending multiple dicts of arrays (e.g. from 
+    multiple input files) into a single dict of arrays 
+    '''
+    if isinstance(val, Mapping):
+        for key2, val2 in val.items() :
+            if key not in sdict :
+                sdict[key] = OrderedDict()
+            append_arrays_dict(key2, val2, sdict[key])
+    else :
+        assert isinstance(val, np.ndarray), "'%s' is not an array, is a %s" % (k, type(v)) 
```

Comment:
seems like this crashes a test because you use "k" and "v" instead of "key" and "value"